src/Analysis/Ast/Test/AssignmentTests.cs

```diff
@@ -319,13 +319,13 @@ class D: ...
         }
 
         [TestMethod, Priority(0)]
-        public async Task AssignAfterClassDef() {
+        public async Task NoClassAssign() {
```

Comment:
Will this undo #1422?

Reply:
Workaround for enum.py unfortunately. See `AssignmentHandler`
```cs
                if(m?.MemberType == PythonMemberType.Class) {
                    // Ignore assignments to classes: enum.py does Enum = None
                    // which prevents persistence from restoring proper type from enum:Enum.
                    continue; 
                }
```

Reply:
Restored